store/memstore/memstore.go

```diff
@@ -34,21 +35,29 @@ func New(maxKeys int) (*MemStore, error) {
 		}
 
 		m = &MemStore{
-			keys: keys,
+			keys:    keys,
+			timeNow: time.Now,
 		}
 	} else {
 		m = &MemStore{
-			m: make(map[string]*int64),
+			m:       make(map[string]*int64),
+			timeNow: time.Now,
 		}
 	}
 	return m, nil
 }
 
+//OverrideTimeNow makes this store use the given function instead of time.Now().
+//This is useful for unit tests that use a simulated wallclock.
```

Comment:
Super nit, but could you please add a space between the `//` and first character for each of these comment lines to make them look like everything else.